include/ensmallen_bits/problems/logistic_regression_function_validation_function.hpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file logistic_regression_function.hpp
+ * @author Marcus Edel
+ *
+ * Minimal implementation of validation function for logisitic regression.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_PROBLEMS_LOGISTIC_REGRESSION_FUNCTION_VALIDATION_FUNCTION_HPP
+#define ENSMALLEN_PROBLEMS_LOGISTIC_REGRESSION_FUNCTION_VALIDATION_FUNCTION_HPP
+
+#include "logistic_regression_function.hpp"
+
+namespace ens {
+namespace test {
+
+using namespace ens::test;
+
+class LogisticRegressionValidationFunction
+{
+ public:
+  LogisticRegressionValidationFunction(LogisticRegression<>& lrIn,
```

Comment:
I'm confused; this class doesn't exist in ensmallen.  I don't think that this will work.  Probably you need to hold a `LogisticRegressionFunction` instead, and then implement the accuracy computation by hand in `Evaluate()`.

Reply:
Actually this class is a wrapper for `LogisticRegressionFunction` in order to provide an `Evaluate` function. However, this class is no longer required since we are using lambda now.